config/locales/idv/en.yml

```diff
@@ -102,8 +101,9 @@ en:
         some basic personal information as well as some financial information from
         you to complete this process. If you don’t have the information we need at
         this time you can continue later.
-      come_back_later: Once your letter arrives, sign into login.gov and enter your
+      come_back_later: Once your letter arrives, sign into login.gov, and enter your
```

Comment:
quick note (looks like this is old copy, but as long as we're here): we've been trying to not hardcode "login.gov" and instead call it `%{app}` and interpolate in the `APP_NAME` constant

Reply:
👍